### PR TITLE
Live deploy improvements

### PR DESCRIPTION
**Pull request by AlasdairGray**
Adding feature requested in [issue 358](https://github.com/BioSchemas/specifications/issues/358) and made various improvements to the table.

Live deploy list now includes link to example page as deployed, structured data testing tool, and rich snippet generation tool. In the future we can add a Bioschemas valdator as well.

This PR also fixes the problem with the table width jumping around.

### CODE REVIEW

File: liveDeploys/elixir.html [hunk 1]
```diff
@@ -29,28 +26,30 @@ <h5>Elixir resources using Bioschema's markup</h5>
             <thead>
                 <tr>
                     <th>Name</th>
-                    <th>Profile</th>
-                    <th>Profile Version</th>
+                    <th class="profile-column">Profile</th>
+                    <th class="structured-data-column">Profile Version</th>
+                    <th class="structured-data-column">Page</th>
```
Comment (ljgarcia) [inline liveDeploys/elixir.html]: Is page a web page? Not sure the column title is the right one

File: liveDeploys/index.html [hunk 2]
```diff
@@ -27,16 +26,18 @@ <h5>{{liveDeploys.description}}</h5>
             <thead>
                 <tr>
                     <th>Name</th>
-                    <th>Profile Version</th>
+                    <th class="structured-data-column">Profile Version</th>
```
Comment (ljgarcia) [inline liveDeploys/index.html]: Do not we need here the profile column as in the previous modified file (elixir.html)?

File: liveDeploys/elixir.html (hunk 1 above)
Comment (AlasdairGray) [reply liveDeploys/elixir.html]: Yes it is the web page with the deployed markup. Happy to rename to web page if you think that is clearer

File: liveDeploys/index.html (hunk 2 above)
Comment (AlasdairGray) [reply liveDeploys/index.html]: No. The live deploys page is grouped by profile in the concertina table. On the elixir page they are grouped by node which is why we need the profile column as well.

File: liveDeploys/elixir.html (hunk 1 above)
Comment (ljgarcia) [reply liveDeploys/elixir.html]: It is probably clear from the content so let's keep it short as it is now

**Comment by ljgarcia** [comment]
@AlasdairGray It looks ready to go

**Pull request merged**